### ISSUE TITLE
Fix errors in console

### ISSUE THREAD

**Issue by brozorec** [Fujicracy/fuji-v2#386]
### Context
There are some errors thrown in the console that don't bother the UI but make it difficult to debug. They are next.js related or MUI related.

Some examples:
![image](https://user-images.githubusercontent.com/9572072/225879321-4c036dcf-64e5-44ba-bee9-ed7e19e73b2e.png)
![image](https://user-images.githubusercontent.com/9572072/225879456-2bdc21aa-8c29-4512-890b-de3f7938074e.png)

### TODOs
- [ ] go through all pages
- [ ] fix the errors if it's a matter of couple minutes of work, otherwise create a ticket

**Comment by NikolaiYurchenko** [comment]
2 of them were fixed [here](https://github.com/Fujicracy/fuji-v2/pull/385/files#diff-beb03b571b56333f0228593b4ccff79d28e0e9a8dfc45b0afb97949cb541a218)
and [here](https://github.com/Fujicracy/fuji-v2/pull/385/files#diff-3bcd13a7922fe1c019cc9e9fffac7d47e5105677dc45c9a22babe2391b4ebb2c)


waiting to be reviewed and merged

**Comment by NikolaiYurchenko** [comment]
working on third

**Comment by ferostar** [comment]
@NikolaiYurchenko we discussed this privately, forgot to add it here. Did you find anything about `[zustand devtools middleware] Please install/enable Redux devtools extension`? I get that warning printed into the console even if I already installed the Chrome extension.

**Comment by NikolaiYurchenko** [comment]
@ferostar this should be only for dev environment since 
```
if (__DEV__ && typeof window !== 'undefined') { 
```
But I`ll search more just to make sure

**Comment by ferostar** [comment]
> @ferostar this should be only for dev environment since
> 
> ```
> if (__DEV__ && typeof window !== 'undefined') { 
> ```
> 
> But I`ll search more just to make sure

Yes, this will never happen on production, it's mostly a convenience for us developers 😅. We always get one of those for each store, so the console gets cluttered. No sense spending much time on this, but please have a look!

**Comment by ferostar** [comment]
Simple issues were addressed, and we now know that `Warning: The result of getSnapshot should be cached to avoid an infinite loop` is caused by `react`.
